cmd/internal/switcher/types/conf.go

```diff
@@ -50,5 +50,58 @@ func (c *Conf) CapitalizeVrfName() {
 	}
 
 	c.Ports.Vrfs = capitalizedVRFs
-	return
+}
+
+func (c *Conf) NewWithoutDownPorts() *Conf {
+	if len(c.Ports.DownPorts) < 1 {
+		return c
+	}
+	newConf := *c
+	newConf.Ports.Vrfs = make(map[string]*Vrf)
```

Comment:
Maybe we can create a deep copy of this object through JSON marshalling as it will be easy to forget modification of this function in the future when new fields are added to the config.

Reply:
What about this comment ?